File: src/main/java/org/neonlabs/e2e/nethereum/model/FaucetRequest.cs
using System.Net.Http;
using System.Text.Json.Serialization;

namespace NeonEndToEnd.org.neonlabs.e2e.nethereum.model
{
  public class FaucetRequest // : HttpContent
  {
    [JsonPropertyName("amount")]
    public int Amount { get; set; }
    [JsonPropertyName("wallet")]
    public string Wallet { get; set; }
  }
}

File: src/main/java/org/neonlabs/e2e/web3j/model/FaucetRequest.java
package org.neonlabs.e2e.web3j.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/** Contains a model. */
@AllArgsConstructor
@Builder
@Data
@NoArgsConstructor
public class FaucetRequest {
    String wallet;
    Integer amount;
}
